packages/client/components/TaskFooterIntegrateMenu.tsx

```diff
@@ -93,6 +137,16 @@ export default createFragmentContainer(TaskFooterIntegrateMenu, {
             ...TaskFooterIntegrateMenuViewerGitHubIntegration @relay(mask: false)
           }
         }
+      }
+      membership: teamMember(userId: null, teamId: $teamId) {
+        integrations {
```

Comment:
let's break this into its own team member fragment so we can repeat it in both places

e.g. 

```gql
fragment TaskFooterIntegrateMenuTeamMember on TeamMember {
  integrations {
atlassian {
            ...TaskFooterIntegrateMenuViewerAtlassianIntegration @relay(mask: false)
          }
          github {
            ...TaskFooterIntegrateMenuViewerGitHubIntegration @relay(mask: false)
          }
  }
}
```